index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+        <header class="main-header"> 
+            <a class="logo" href="#">MHabert</a>
+            <nav class="nav-container">
+                <a href="#" class="menu"><img src="menu.png"page logo=""></a>
+                
+        </nav>
+    </header>
+
+
+    <div  class="container">
```

Comment:
[OPTIONAL]
- [ ] Kindly remove this `div` tag, it would be nice to use a `main` tag instead for semantics reasons.

Reply:
[OPTIONAL]
- [ ] I strongly recommend you to take all the optional comments into consideration as they can make your code better.